### DISCUSSION TITLE
Explanation of the display under actions

### DISCUSSION THREAD

**Discussion by KodaCHC** [community/community#25752] [Actions]
<p>I create 10 commits and push them to github. No action is executed in the process.</p>
<p>Now I create a new tag and push it.</p>
<pre><code class="lang-auto">git tag -a v1.0.0 -m "New release for v1.0.0" &amp;&amp; git push origin 1.0.0
</code></pre>
<p>My action on GitHub</p>
<pre><code class="lang-auto">My 5th commit
build package #30: Commit xxxxx pushed by username
</code></pre>
<p>But now I have two questions. For me the action display is a bit confusing</p>
<ul>
<li>Why is my last commit shown in the headline and not, for example, my commit from the created tag?</li>
<li>Why do I only see the latest commit in the description? I create a release for the newly created tag with the action</li>
</ul>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="KodaCHC" data-post="1" data-topic="221951">
<div class="title">
<div class="quote-controls"></div>
 KodaCHC:</div>
<blockquote>
<p><code>git tag -a v1.0.0 -m "New release for v1.0.0" &amp;&amp; git push origin 1.0.0</code></p>
</blockquote>
</aside>
<p>The tag and the pushed ref aren’t the same (<code>v1.0.0</code> vs. <code>1.0.0</code>). Other than it’s hard to say what exactly is going on without more details, including how the workflow is triggered.</p>

**Comment by KodaCHC** [comment]
<p>Sorry that was a mistake when writing. it is also v1.0.0</p>
<p>My workflow is started as follows</p>
<pre><code class="lang-auto">name: build release
on:
  push:
    tags:
      - '*'
jobs:
  build:
</code></pre>

**Comment by airtower-luna** [comment]
<p>In that case the commit ID should be that of the tagged commit, you check that with <code>git show v1.0.0</code>, and the tag should be shown to the right of that label.</p>

**Comment by KodaCHC** [comment]
<p>GitHub shows the commit and not the tag release. I have created an example here:<br>
My Action: <a href="https://github.com/KodaCHC/MyFirstAction" class="inline-onebox" rel="noopener nofollow ugc">GitHub - KodaCHC/MyFirstAction</a><br>
My Repository: <a href="https://github.com/KodaCHC/MyFirstRepository" class="inline-onebox" rel="noopener nofollow ugc">GitHub - KodaCHC/MyFirstRepository</a></p>
<p>My steps</p>
<pre><code class="lang-auto">git clone https://github.com/KodaCHC/MyFirstRepository.git
cd MyFirstRepository/
echo "empty" &gt; file2.txt
git add -A &amp;&amp; git commit -m "File 2" &amp;&amp; git push -f # Commit created: 62ea5d48365c77a28c960417183f090c410d7eee
echo "empty" &gt; file3.txt
git add -A &amp;&amp; git commit -m "File 3" &amp;&amp; git push -f # Commit created: 138659b018ffd9ba48229dcdf92379bc3140fe9a
git tag -a v1.0.0 -m "New release for v1.0.0" &amp;&amp; git push origin v1.0.0
</code></pre>
<p>Now the Action show me the headline of my last commit “File 3”</p>
<pre><code class="lang-auto">File 3
My Action #1: Commit 138659b pushed by KodaCHC
</code></pre>

**Comment by airtower-luna** [comment, marked as answer]
<p>This is that I see:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/d/5df352e284172abf5d0fa20eb0e50f0d32f99278.png" data-download-href="/uploads/short-url/dp7IRdBPfIEdkAE7LVcmTIWfSOs.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/5814919/181093046-d01142f8-110a-4fd4-8e20-e321b5992ac8.png" alt="image" data-base62-sha1="dp7IRdBPfIEdkAE7LVcmTIWfSOs" width="690" height="259" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5df352e284172abf5d0fa20eb0e50f0d32f99278_2_690x259.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5df352e284172abf5d0fa20eb0e50f0d32f99278_2_1035x388.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/d/5df352e284172abf5d0fa20eb0e50f0d32f99278.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5df352e284172abf5d0fa20eb0e50f0d32f99278_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1089×409 39.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Looks correct to me. The tag name is to the right, the blue link (the ref that was pushed). The “File 3” is the title of the commit on which the workflow ran (what’s called <code>GITHUB_SHA</code> in the <a href="https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#push">event documentation</a>).</p>

**Comment by KodaCHC** [comment]
<p>For me the confusing thing is only why the last commit is shown. I would have thought the comment of the new tag is used.</p>
